src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/NonAssociation.kt

```diff
@@ -13,15 +13,17 @@ data class NonAssociation(
 
   @Schema(description = "Prisoner number to not associate", required = true, example = "A1234BC")
   val firstPrisonerNumber: String,
-  @Schema(description = "Reason why this prisoner should be kept apart from the other", required = true, example = "VICTIM")
-  val firstPrisonerReason: NonAssociationReason,
+  @Schema(description = "This prisoner’s role in the non-association", required = true, example = "VICTIM")
+  val firstPrisonerRole: Role,
   @Schema(description = "Prisoner number to not associate", required = true, example = "D5678EF")
   val secondPrisonerNumber: String,
-  @Schema(description = "Reason why this prisoner should be kept apart from the other", required = true, example = "PERPETRATOR")
-  val secondPrisonerReason: NonAssociationReason,
+  @Schema(description = "Other prisoner’s role in the non-association", required = true, example = "PERPETRATOR")
+  val secondPrisonerRole: Role,
 
+  @Schema(description = "Reason why these prisoners should be kept apart", required = true, example = "BULLYING")
+  val reason: Reason,
```

Comment:
The "reason" for a non-association applies to the whole thing, not either person involved.